napari-workshops/notebooks/manual_annotation.md

```diff
@@ -20,8 +20,7 @@ There are 3 main types of manual annotation that napari provides, each correspon
 - drawing 2D polygons to identify particular regions of interest with the **Shapes** layer
 - painting labels to provide a pixel-wise annotation of an image with the **Labels** layer
 
-This tutorial will explore these three manual annotations in **[napari](https://napari.org/)**, using that same data from the image visualization tutorial.
-
+This tutorial will explore these three manual annotations in napari, using the same data from the previous image visualization tutorial. Once again, we will try to highlight the bidirectional communication between the viewer and the Python kernel.
```

Comment:
```suggestion
This tutorial will explore these three manual annotations in napari, using the same data from the [Bioimage visualization in Python tutorial](intro_bioimage_visualization.md). Once again, we will try to highlight the bidirectional communication between the viewer and the Python kernel.
```

I think this is the right link? People may not be reading these notebooks in order.